build.sbt

```diff
@@ -11,7 +11,10 @@ crossScalaVersions := Seq("2.11.12", "2.12.4")
 libraryDependencies += guice
 libraryDependencies += "com.typesafe.play" %% "play-slick" % "3.0.3"
 libraryDependencies += "com.typesafe.play" %% "play-slick-evolutions" % "3.0.3"
+libraryDependencies += "org.scalatestplus.play" %% "scalatestplus-play" % "3.1.2" % "test"
 
 libraryDependencies += "com.h2database" % "h2" % "1.4.197"
 
 libraryDependencies += specs2 % Test
+
+javaOptions in Test += "-Dconfig.file=conf/test.conf"
```

Comment:
If `test.conf` is not necessary, we must also remove this line.

Reply:
@marcospereira yes, You are right. It is not necessary. 
For new users, it is easy to have this. They will keep all test configuration in test.conf. If they are using `sbt test ` all the configuration will come from test.conf.